.vscode/settings.json

```
@@ -29,4 +29,13 @@
     "github-actions.remote-name": "upstream",
     // NOTE: This disable the plugins formatting so astyle is used.
     "C_Cpp.formatting": "disabled",
+    "emeraldwalk.runonsave": {
+        "commands": [
+            {
+                "match": "\\.json$",
+                "isAsync": true,
+                "cmd": "${workspaceFolder}/tools/format/json_formatter ${file}"
+            },
+        ]
```

On linux the formatter gets .cgi extension, seems the extension silently executes them both so probably ok to have both
```suggestion
        "commands": [
            {
                "match": "\\.json$",
                "isAsync": true,
                "cmd": "${workspaceFolder}/tools/format/json_formatter ${file}",
            },
            {
                "match": "\\.json$",
                "isAsync": true,
                "cmd": "${workspaceFolder}/tools/format/json_formatter.cgi ${file}",
            },
        ]
```